docs/_source/getting_started/installation/deployments/docker-quickstart.md

```diff
@@ -1,9 +1,8 @@
 # Docker Quickstart
 
-Argilla is composed of a Python Server with Elasticsearch as the database layer, and a Python Client to create and
-manage datasets.
+Argilla is composed of a Python Server with Elasticsearch as the database layer, and a Python Client to create and manage datasets.
```

Comment:
Now we also support user and workspace management, so maybe it's nice to include this here too!

```suggestion
Argilla is composed of a Python Server with Elasticsearch as the database layer, and a Python Client to create and manage datasets, users, and workspaces.
```